src/pages/index.js

```diff
@@ -56,7 +56,9 @@ function Landing() {
 `
 # Helm chart
 helm repo add s3gw https://aquarist-labs.github.io/s3gw-charts/
-helm install s3gw s3gw/s3gw --namespace s3gw-system --create-namespace
+helm install s3gw s3gw/s3gw --namespace s3gw-system --create-namespace \\
+    --set publicDomain=YOUR_IP_ADDRESS.sslip.io \\
+    --set ui.publicDomain=YOUR_IP_ADDRESS.sslip.io
```

Comment:
I think we've been avoiding using this sort of addresses in examples, and in code as well. I think it's okay to suggest it in a different paragraph, something like "for testing purposes you might want to check it out using `YOUR_IP_ADDRESS.sslip.io`". But for general documentation it might send the wrong message. And in the front page it might even make it weirder.

Reply:
Fair enough - I'm fine to not use sslip.io - I just wanted to be able to put _something_ there to show that publicDomain does need to be set in order for `helm install` to work. Would `--set publicDomain=YOUR_DOMAIN_NAME` be OK in the example code?

Reply:
Cool, done :-)

I didn't add anything about sslip.io to the extra paragraph below "getting started" as IMO that's already long enough for a brief intro page. We can always look at adding that into the docs somewhere as a suggestion for testing.

Reply:
sounds good! :)